toolchain/lower/function_context.h

```diff
@@ -45,6 +46,8 @@ class FunctionContext {
 
   // Returns a value for the given instruction.
   auto GetValue(SemIR::InstId inst_id) -> llvm::Value* {
+    // TODO: if(specific_id_.has_value()) may need to update inst_id first.
```

Comment:
could this be a CARBON_FATAL TODO sort of thing? (so if someone does write a test the fail, rather than getting some weird behavior later on?)

Reply:
The change here looks more involved to get this to work. I'd rather leave this as is in the PR and build on it in the next iteration, if you're ok with that.